spec/features/user_deletes_cart_items_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+feature 'delete cart items' do
+
+  before(:each) do
+    @item = Item.create(title: "Eat Cheese and Die",
+      description: "The spookiest grilled cheese.",
+      price: 50.65)
+    @item2 = Item.create(title: "It Came From Beneath the Sink",
+      description: "You don't want to know.",
+      price: 200.99)
+
+    visit item_path(@item)
+    click_on "Add to cart"
+  end
+
+  scenario 'delete an item' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.